commit 369d595d95f4945f0e14dd02757cf64eaf8512d9
Author: Willy Tarreau <w@1wt.eu>
Date:   Fri Mar 29 17:40:23 2019 +0100

    BUILD: makefile: fix build of IPv6 header on aix51
    
    ip6_hdr is called ip6hdr there and is only defined when STEVENS_API is
    defined.
    
    (cherry picked from commit 6f4fd1b183fc70863926bdc31d7f0b4739c92c56)
    Signed-off-by: Willy Tarreau <w@1wt.eu>
    (cherry picked from commit 99903a0c6419edb515a07a6e1fba4b8bc78babd1)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/Makefile b/Makefile
index bdf5a2d0..81e70e02 100644
--- a/Makefile
+++ b/Makefile
@@ -358,7 +358,7 @@ ifeq ($(TARGET),aix51)
   # This is for AIX 5.1
   USE_POLL        = implicit
   USE_LIBCRYPT    = implicit
-  TARGET_CFLAGS   = -Dss_family=__ss_family
+  TARGET_CFLAGS   = -Dss_family=__ss_family -Dip6_hdr=ip6hdr -DSTEVENS_API
   DEBUG_CFLAGS    =
 else
 ifeq ($(TARGET),aix52)
